source/write-scripts.txt

```diff
@@ -0,0 +1,120 @@
+.. _mdb-shell-write-scripts:
+
+=======================================
+Write Scripts for {+mdb-shell+}
+=======================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can write scripts for the MongoDB Shell that manipulate data in
```

Comment:
Source constant?
```suggestion
You can write scripts for the {+mdb-shell+} that manipulate data in
```

Reply:
replaced with substitution in write-scripts.txt and connect.txt